pkg/pillar/cmd/upgradeconverter/upgradeconverterutils.go

```diff
@@ -0,0 +1,50 @@
+package upgradeconverter
+
+import (
+	"encoding/json"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+// FilePathData - Stores old and new file paths
+type FilePathData struct {
+	OldVersionPath string
+	NewVersionPath string
+}
+
+func fileExists(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
+
+func fileTimeStamp(filename string) (time.Time, error) {
+	file, err := os.Stat(filename)
+	if err != nil {
+		return time.Now(), err
```

Comment:
Why now? It's an error condition

Reply:
What should I return instead? I return the error, I thought that was sufficent.